.github/CODE_OF_CONDUCT_EN.md

```diff
@@ -19,13 +19,13 @@ version 1.0, 2020/3/3
 
 ## International
 
-* We will design this website be intuitive to use and understand.
+* We will design this website to be intuitive for use and easy to understand.
 * We will provide the information in multiple languages.
 
 ## Be open
 
-* Open source: We will release most of our source codes and knowledge used to develop this website for free, to encourage other local authorities to utilise them.
-* Open data: We will release the data for free in reusable data form, which is available for everyone.
+* Open source: We will release most of our source code and knowledge used to develop this website for free of charge, in order to encourage other local authorities to utilise them to the full extent.
```

Comment:
Either "for free" or "free of charge" sounds natural, but "for free of charge" is a bit unusual.

Reply:
Ur completely right my mistake